core/components/atoms/metricInput/MetricInput.tsx

```diff
@@ -284,7 +283,7 @@ export const MetricInput = React.forwardRef<HTMLInputElement, MetricInputProps>(
       {suffix && (
         <Text
           data-test="DesignSystem-MetricInput--suffix"
-          className={size === 'regular' ? 'ml-4' : 'ml-5'}
+          className={size === 'regular' ? 'ml-4 mr-3' : 'ml-5 mr-5'}
```

Comment:
Please specify all the conditional css classes using `classNames`. Also instead of writing `ml-5 mr-5` separately we can directly use `mx-5`

Reply:
Done.